### PR TITLE
gff3toembl: Converter for going from GFF3 to an EMBL submission format (new formula)

### PR DESCRIPTION
**Pull request by andrewjpage**
Takes a GFF3 file (such as those created by prokka @torstenseemann ) and converts it to a format valid for submission to EMBL.  This isnt a generic converter. Data thats not valid for EMBL gets stripped out or modified to make it valid.
This depends on the GenomeTools python library, which is exposed with this pull request:
https://github.com/Homebrew/homebrew-science/pull/2524

### CODE REVIEW

File: gff3toembl.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Gff3toembl < Formula
+  desc "Convert GFF3 files to a format suitable for submission to EMBL"
+  homepage "https://github.com/sanger-pathogens/gff3toembl"
+  url "https://github.com/sanger-pathogens/gff3toembl/archive/v1.0.0.tar.gz"
+  sha256 "a224e5406098714a12d257e307239120150e80b8b75fcf2f77149ed034757f38"
+  head "https://github.com/sanger-pathogens/gff3toembl.git"
+  # tag "bioinformatics"
+
+  depends_on :python
```
Comment (sjackman) [inline gff3toembl.rb]: See comments in https://github.com/Homebrew/homebrew-science/pull/2524#discussion_r34816387

**Comment by sjackman** [comment]
```
Error Message

failed: brew audit Homebrew/science/gff3toembl --strict --online
Stacktrace

        ==> brew style gff3toembl

1 file inspected, no offenses detected
Error: 2 problems in 1 formula

==> audit problems
homebrew/science/gff3toembl:
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as homebrew/science/gff3toembl + desc. (currently 91)
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
```

**Comment by sjackman** [comment]
These warnings can be ignored.

**Comment by tseemann** [comment]
*"Github repository not notable enough"* - Burn! Brew is getting personal now! ;-)

**Comment by tseemann** [comment]
@BrewStatusBot test this please

**Comment by tseemann** [comment]
@BrewTestBot test this please

**Comment by sjackman** [comment]
http://bot.brew.sh/job/Homebrew%20Science%20Pull%20Requests/2691/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_Homebrew_science_gff3toembl___strict___online/
```
Error Message

failed: brew test --verbose Homebrew/science/gff3toembl
Stacktrace

        Testing homebrew/science/gff3toembl
==> gff3_to_embl -h 2>&1
Error: homebrew/science/gff3toembl: failed
<0> expected but was
<1>.
```

File: gff3toembl.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class Gff3toembl < Formula
+  desc "Convert GFF3 files to a format suitable for submission to EMBL"
+  homepage "https://github.com/sanger-pathogens/gff3toembl"
+  # tag "bioinformatics"
+
+  url "https://github.com/sanger-pathogens/gff3toembl/archive/v1.1.0.tar.gz"
+  sha256 "f5ec9e08660519be6be22b487c9acceabb629bb19c8e45bc09df74d15b2bdae7"
+  head "https://github.com/sanger-pathogens/gff3toembl.git"
+
+  depends_on :python
```
Comment (dpo) [inline gff3toembl.rb]: `if MacOS.version <= :snow_leopard`

**Comment by sjackman** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
@andrewjpage since `gff3_to_embl -h` returns `1` consider using the 2 argument form of `shell_output`. Run `cd $(brew --repo homebrew/core ) && grep shell_output -r .` for examples. If there's a published paper for this consider adding a `# doi ...` tag. Finally please fix these audit warnings:

```
homebrew/science/gff3toembl:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
  * fully scope test shell_output calls e.g. shell_output "#{bin}/gff3_to_embl"
```

**Comment by iMichka** [comment]
Closing here as there was no activity since a few months. If somebody really needs this a new PR can be opened.

**Comment by sjackman** [comment]
@andrewjpage Any interest in reviving this PR?

**Pull request closed without merging**